turbinia/workers/analysis/ssh_analyzer.py

```diff
@@ -0,0 +1,478 @@
+    """Return priority value
+    
+    Args:
+      priority_string(str): Priority values as string e.g. HIGH, MEDIUM, LOW
+    
+    Returns:
+      Priority: Returns priority value of priority_string.
+    """
+    analyzer_priority_string = priority_string.upper()
+
+    try:
+      return Priority[analyzer_priority_string]
+    except KeyError:
+      log.error(
+          'Priority %s does not exist. Returning LOW', analyzer_priority_string)
+      return Priority.LOW
+
+  def brute_force_analysis(self, df: pd.DataFrame) -> Tuple[Priority, str, str]:
+    """Run brute force analysis.
+    
+    Args:
+      df (pd.DataFrame): Pandas dataframe of SSH events.
+    
+    Returns:
+      Tuple[Priority, str, str]: Returns brute force analysis result as tuple.
+        Priority: Priority of the findings.
+        str: Brief summary of the findings.
+        str: Detailed information as markdown. 
+    """
+    bfa = BruteForceAnalyzer()
+
+    bfa_result = bfa.run(df)
+    if not bfa_result:
+      return (
+          Priority.LOW, 'No findings for brute force analysis',
+          '## Brute force analysis\n\n- No findings')
+
+    result_priority = bfa_result.result_priority
+    result_summary = bfa_result.result_summary
+    result_markdown = bfa_result.result_markdown
+
+    priority = self.get_priority_value(result_priority)
+
+    if not result_summary:
+      result_summary = 'No findings for brute force analysis'
+    if not result_markdown:
+      result_markdown = '## Brute Force Analysis\n\n- No findings'
+
+    return (priority, result_summary, result_markdown)
+
+  def run(
+      self, evidence: Evidence,
+      result: TurbiniaTaskResult) -> TurbiniaTaskResult:
+    """Run the SSH Auth Analyzer worker.
+
+    Args:
+      evidence (Evidence object): The evidence being processed by analyzer.
+      result (TurbiniaTaskResult): The object to place task results into.
+    
+    Returns:
+      TurbiniaTaskResult object.
+    """
+
+    # Output file and evidence
+    output_file_name = 'linux_ssh_analysis.md'
+    output_file_path = os.path.join(self.output_dir, output_file_name)
+    output_evidence = ReportText(source_path=output_file_path)
+
+    # Analyzer outputs
+    analyzer_output_priority = Priority.LOW
+    analyzer_output_summary = ''
+    analyzer_output_report = ''
+    output_summary_list = []
+    output_report_list = []
+
+    try:
+      collected_artifacts = extract_artifacts(
+          artifact_names=['LinuxAuthLogs'], disk_path=evidence.local_path,
+          output_dir=self.output_dir, credentials=evidence.credentials)
+      result.log(f'collected artifacts: {collected_artifacts}')
+    except TurbiniaException as exception:
+      result.close(self, success=False, status=str(exception))
+      return result
+
+    log_dir = os.path.join(self.output_dir, 'var', 'log')
+    result.log(f'Checking log directory {log_dir}')
+
+    #if not os.path.exists(log_dir):
```

Comment:
Is there a reason this is commented out?  Should we uncomment it or remove it?

Reply:
Fixed - uncommented the block.